.circleci/config.yml

```
@@ -46,16 +46,17 @@ jobs:
       go_version:
         type: string
     environment:
-      CI_E2E_FILENAME: "fa6ad40d/rel-nightly"
+      CI_E2E_FILENAME: "f9acfc35/rel-nightly"
```

updating to a more recent test

---

more recent, this definitely has the new txn-counter changes
```suggestion
      CI_E2E_FILENAME: "f99e7b0c/rel-nightly"
```